### PR TITLE
Improve EmitConstants enums

### PR DESCRIPTION
**Pull request by myoung301**
-Remove all functions that retrieve the enums and include them directly in EmitConstants module
-Get auto-complete working

### CODE REVIEW

**Review by apollo-crum — APPROVED** [review]
Looks a lot cleaner. :-)

**Review by MaxJPRey — APPROVED** [review]
@myoung301 This is great to use Enums to access those constant values! Thanks.

File: pyaedt/emit_core/__init__.py [hunk 1]
```diff
@@ -32,3 +62,4 @@ def _set_api(aedt_version):
         global EMIT_API_PYTHON
         EMIT_API_PYTHON = import_module("EmitApiPython")
         logger.info("Loaded {}".format(EMIT_API_PYTHON.EmitApi().get_version(True)))
+    _init_enums()
```
Comment (jsalant22) [inline pyaedt/emit_core/__init__.py:65]: @myoung301 Don't the enums require v232 or newer? This looks like its outside the `if`

File: pyaedt/emit_core/__init__.py [hunk 2]
```diff
@@ -12,10 +16,36 @@ def emit_api_python():
     """
     Get the Emit backend API.
```
Comment (PipKat) [inline pyaedt/emit_core/__init__.py:17]:
```suggestion
    Get the EMIT backend API.
```

File: pyaedt/emit_core/__init__.py [hunk 3]
```diff
@@ -12,10 +16,36 @@ def emit_api_python():
...
 
+    The backend API is available once a pyaedt.Emit() object has been created. An Exception is raised if this method is called before a pyaedt.Emit() object has been created.
```
Comment (PipKat) [inline pyaedt/emit_core/__init__.py:19]:
```suggestion
    The backend API is available once a pyaedt.Emit() object has been created. An exception is raised if this method is called before a ``pyaedt.Emit()`` object has been created.
```

File: pyaedt/emit_core/emit_constants.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
+"""
+Enums from EmitApiPython are defined as none until the EmitApiPython module initializes.
+This allows IDE auto-complete to find them and emit_constants to import before the
+EmitApiPython module has loaded (typically when a pyaedt.Emit() object is created).
```
Comment (PipKat) [inline pyaedt/emit_core/emit_constants.py:4]:
```suggestion
Enums from the ``EmitApiPython`` module are defined as ``None`` until this module initializes.
This allows IDE auto-complete to find them and ``emit_constants`` to import before the
``EmitApiPython`` module has loaded (typically when a ``pyaedt.Emit()`` object is created).
```

File: pyaedt/emit_core/__init__.py (hunk 1 above)
Comment (myoung301) [reply pyaedt/emit_core/__init__.py]: @jsalant22 I applied the change.

**Pull request merged**